spec/automation/unit/method_validation/calculate_limits_spec.rb

```diff
@@ -2,11 +2,17 @@
 include QuotaHelper
 
 describe "Quota Validation" do
-  def run_automate_method(provision_request)
+  def run_automate_method(provision_request, quota_source = 'tenant')
```

Comment:
@tinaafitz Overall this is good.  The one thing I am thinking is quota_source should be explicitly passed from the test instead of defaulted.